lib/utils.js

```diff
@@ -66,4 +69,22 @@ async function killAppUsingAppName (udid, appName) {
   }
 }
 
-export { detectUdid, getAndCheckXcodeVersion, getAndCheckIosSdkVersion, killAppUsingAppName };
+async function adjustWDAAttachmentsPermissions(perms) {
+  let processedFolders = [];
+  let derivedDataSearchMask = path.join(process.env['HOME'], WDA_DERIVED_DATA_SEARCH_SUFFIX);
```

Comment:
@jlipps It will do. The thing is, that this method is not going to be called anywhere except of MacOS. the problem with const declaration was, that all constant values are calculated in any case, but things inside method calls only do it by demand (e.g explicit method call).